Crash when clicking on the help dialog button during login

When testing [this PR](https://github.com/woocommerce/woocommerce-android/pull/3500), I found the following crash when clicking on the `Find my Site address` -> `Need more Help?` button when entering my site address during login.

```
2021-02-01 10:18:44.310 4206-4206/com.woocommerce.android.prealpha E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.woocommerce.android.prealpha, PID: 4206
    android.view.InflateException: Binary XML file line #20 in com.woocommerce.android.prealpha:layout/support_email_and_name_dialog: Binary XML file line #20 in com.woocommerce.android.prealpha:layout/support_email_and_name_dialog: Error inflating class com.woocommerce.android.widgets.WCMaterialOutlinedEditTextView
    Caused by: android.view.InflateException: Binary XML file line #20 in com.woocommerce.android.prealpha:layout/support_email_and_name_dialog: Error inflating class com.woocommerce.android.widgets.WCMaterialOutlinedEditTextView
    Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Constructor.newInstance0(Native Method)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
        at android.view.LayoutInflater.createView(LayoutInflater.java:852)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1004)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:959)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:1121)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:680)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:532)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:479)
        at com.woocommerce.android.support.SupportHelperKt.supportIdentityInputDialogLayout(SupportHelper.kt:95)
        at com.woocommerce.android.support.SupportHelperKt.access$supportIdentityInputDialogLayout(SupportHelper.kt:1)
        at com.woocommerce.android.support.SupportHelper.showSupportIdentityInputDialog(SupportHelper.kt:36)
        at com.woocommerce.android.support.SupportHelper.showSupportIdentityInputDialog$default(SupportHelper.kt:32)
        at com.woocommerce.android.support.ZendeskHelper.requireIdentity(ZendeskHelper.kt:292)
        at com.woocommerce.android.support.ZendeskHelper.createNewTicket(ZendeskHelper.kt:156)
        at com.woocommerce.android.support.ZendeskHelper.createNewTicket$default(ZendeskHelper.kt:151)
        at com.woocommerce.android.ui.login.LoginActivity.helpFindingSiteAddress(LoginActivity.kt:445)
        at org.wordpress.android.login.LoginSiteAddressHelpDialogFragment$2.onClick(LoginSiteAddressHelpDialogFragment.java:68)
        at androidx.appcompat.app.AlertController$ButtonHandler.handleMessage(AlertController.java:167)
        at android.os.Handler.dispatchMessage(Handler.java:106)
```

This is not happening in the release version so I suspect it is due to the recent changes to Viewbinding in [this PR](https://github.com/woocommerce/woocommerce-android/pull/3498).

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/widgets/WCMaterialOutlinedCurrencyEditTextView.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/widgets/WCMaterialOutlinedCurrencyEditTextView.kt
@@ -70,12 +70,6 @@ class WCMaterialOutlinedCurrencyEditTextView @JvmOverloads constructor(
         binding.currencyEditText.setValue(currentValue)
     }
 
-    override fun setEnabled(enabled: Boolean) {
-        super.setEnabled(enabled)
-
-        binding.currencyEditText.isEnabled = enabled
-    }
-
     fun getCurrencyEditText(): CurrencyEditText = binding.currencyEditText
 
     override fun onSaveInstanceState(): Parcelable? {

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/widgets/WCMaterialOutlinedCurrencyEditTextView.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/widgets/WCMaterialOutlinedCurrencyEditTextView.kt
@@ -6,7 +6,6 @@ import android.os.Parcelable
 import android.util.AttributeSet
 import android.util.SparseArray
 import android.view.LayoutInflater
-import android.view.View
 import androidx.annotation.AttrRes
 import androidx.lifecycle.LiveData
 import com.google.android.material.textfield.TextInputLayout
@@ -39,7 +38,6 @@ class WCMaterialOutlinedCurrencyEditTextView @JvmOverloads constructor(
         private const val KEY_SUPER_STATE = "WC-OUTLINED-CURRENCY-VIEW-SUPER-STATE"
     }
     init {
-        View.inflate(context, R.layout.view_material_outlined_currency_edittext, this)
         if (attrs != null) {
             val a = context.obtainStyledAttributes(
                     attrs,

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/widgets/WCMaterialOutlinedEditTextView.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/widgets/WCMaterialOutlinedEditTextView.kt
@@ -110,12 +110,6 @@ class WCMaterialOutlinedEditTextView @JvmOverloads constructor(
         binding.editText.filters += InputFilter.LengthFilter(max)
     }
 
-    override fun setEnabled(enabled: Boolean) {
-        super.setEnabled(enabled)
-
-        binding.editText.isEnabled = enabled
-    }
-
     override fun onSaveInstanceState(): Parcelable? {
         val bundle = Bundle()
         binding.editText.onSaveInstanceState()?.let {

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/widgets/WCMaterialOutlinedSpinnerView.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/widgets/WCMaterialOutlinedSpinnerView.kt
@@ -59,12 +59,6 @@ class WCMaterialOutlinedSpinnerView @JvmOverloads constructor(
 
     fun getText() = binding.spinnerEditText.text.toString()
 
-    override fun setEnabled(enabled: Boolean) {
-        super.setEnabled(enabled)
-
-        binding.spinnerEditText.isEnabled = enabled
-    }
-
     override fun onSaveInstanceState(): Parcelable? {
         val bundle = Bundle()
         binding.spinnerEditText.onSaveInstanceState()?.let {

